packages/echo-base/src/module/utils.ts

```diff
@@ -1,19 +1,72 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import { AppMetadata, SingleApp } from '../types/module';
+import { EchoModuleApiCreator, ModuleRequester } from '../types/creators';
+import { EchoModule, ModuleMetaData } from '../types/module';
 
+/**
+ *
+ *
+ * @export
+ * @param {string} id
+ */
 export function removeElementById(id: string): void {
     document.getElementById(id)?.remove();
 }
 
-export function isfunc(f: any): f is Function {
+/**
+ * Helper function for Checking if function is present.
+ *
+ * @param {EchoModuleApiCreator} createModuleApi
+ * @return {*}  {boolean}
+ */
+export function checkFunction(func: EchoModuleApiCreator | ModuleRequester, errorMessage: string): boolean {
+    if (!isfunc(func)) {
+        console.warn(errorMessage);
+        return false;
+    }
+
+    return true;
+}
+
+/**
+ *
+ *
+ * @export
+ * @param {*} f
+ * @return {*}  {f is Function}
+ */
+export function isfunc(f: unknown): f is Function {
     return typeof f === 'function';
 }
 
-export function createEmptyModule(meta: AppMetadata): SingleApp {
+/**
+ *
+ *
+ * @export
+ * @param {ModuleMetaData} meta
+ * @return {*}  {SingleApp}
+ */
+export function createEmptyApp(meta: ModuleMetaData): EchoModule {
     return {
         ...meta,
         setup(): void {
             // Empty module
         }
     };
 }
+
+export function isProduction(): boolean {
```

Comment:
hva er forskjell på denne, og Env().isProduction som du har i en annen fil? Fører raskt til bugs med forskjellige måter å sjekke dette på. Men kanskje env() er tilgjengelig for modules

Reply:
slipper core og skal bare brukes i base ;)

Reply:
Husk denne ikke vil fungere i azure, hvis dere ikke har gjort noe her da.
Er derfor vi bruker env() i echopedia
Er litt redd denne isProduction her fører til misforståelser og bugs senere, ved at folk bruker feil

Reply:
denne er lik i ecopedia